form-interaction.rst

```diff
@@ -0,0 +1,128 @@
+Interaction of constraints, relevants and read only
```

Comment:
It would probably be good to link to the relevant info at xlsform.org, for each section.

Something simple like:  
[Read more about constraints in the XLSForm documentation.](http://xlsform.org)  
etc...

Reply:
I added a note for this after explanation of all sections.

Reply:
A note at the bottom of the "Calculation" section doesn't _really_ refer to the five sections preceding.

It would be better to put it where it is relevant, and use section links. 

 - A link to [constraints docs](http://xlsform.org/#constraints) where you discuss constraints. 
 - A link to the [relevants docs](http://xlsform.org/#relevant) where you talk about relevants. 
 - Etc..